modules/tensor_mechanics/doc/content/source/materials/ComputeFiniteStrain.md

```diff
@@ -5,23 +5,74 @@
 \end{equation}
 where $\hat{\boldsymbol{u}}(\boldsymbol{x})$ is the incremental displacement field for the time step, and
-$\boldsymbol{x}$ is the position vector of materials points in $\kappa_{n+1}$. Note that
-$\hat{\boldsymbol{F}}$ is NOT the deformation gradient, but rather the incremental deformation gradient
-of $\kappa_{n+1}$ with respect to $\kappa_n$. Thus, $\hat{\boldsymbol{F}} = \boldsymbol{F}_{n+1}
+$\boldsymbol{x}_{n+1}$ is the position vector of materials points in $\kappa_{n+1}$.
+
+!alert! note title=Incremental vs Total Deformation Gradient
+Note that $\hat{\boldsymbol{F}}$ is NOT the deformation gradient, but rather the incremental deformation gradient
+of $\kappa_{n+1}$ with respect to $\kappa_n$. Thus $\hat{\boldsymbol{F}} = \boldsymbol{F}_{n+1}
 \boldsymbol{F}_n^{-1}$, where $\boldsymbol{F}_n$ is the total deformation gradient at time $t_n$.
+!alert-end!
+
+As shown in [cite:zhang2018modified], the incremental deformation gradient can
+be multiplicatly decomposed into an incremental rotation tensor, $\boldsymbol{\hat{R}}$,
+and the incremental right stretch tensor, $\boldsymbol{\hat{U}}
+\begin{equation}
+  \boldsymbol{\hat{F}} = \boldsymbol{\hat{R}} \cdot \boldsymbol{\hat{U}}
+\end{equation}
+The incremental right Cauchy-Green deformation tensor, $\boldsymbol{\hat{C}}$,
+can be given in terms of $\boldsymbol{\hat{U}}$ by subsituting the polar decomposition
+of the deformation gradient into the classical definition for $\boldsymbol{\hat{C}}$.
+\begin{equation}
+  \label{eqn:right_green_cauchy_deformation_tensor}
+  \boldsymbol{\hat{C}} = \boldsymbol{\hat{F}}^T \cdot \boldsymbol{\hat{F}} = \boldsymbol{\hat{U}}^T \cdot \boldsymbol{\hat{R}}^T \cdot \boldsymbol{\hat{R}} \cdot \boldsymbol{\hat{U}} = \boldsymbol{\hat{U}}^2
+\end{equation}
+where the orthogonal nature of $\boldsymbol{\hat{R}}$ enables the simplification
+given above. Thus $\boldsymbol{\hat{U}}$ can be computed from $\boldsymbol{\hat{C}}$ as
+\begin{equation}
+  \label{eqn:definition_stretch_tensor}
+  \boldsymbol{\hat{U}} = \boldsymbol{\hat{C}}^\frac{1}{2}
+\end{equation}
+which can be evaluated by performing a spectral decomposition of $\boldsymbol{\hat{C}}$.
+Once $\boldsymbol{\hat{U}}$ has been computed, the multiplicative decomposition
+of the deformation graidient is used to find the incremental rotation tensor
+$\boldsymbol{\hat{R}}$ and the stretching rate $\boldsymbol{D}$.
+Following [cite:rashid1993incremental], the stretching rate tensor can be expressed in terms
+of 'incremental' right Cauchy-Green deformation tensor
+\begin{equation}
+  \label{eqn:stretching_tensor_definition}
+  \boldsymbol{D} = \frac{1}{\Delta t}\log({\hat{\boldsymbol{C}}^{1/2}})
+\end{equation}
+
+This incremental streteching rate tensor can then be used as the work conjugate
+for a stress measure, as shown in [cite:zhang2018modified], or used to find a strain
+measure. Therefore `ComputeFiniteStrain` includes two different methods to solve
```

Comment:
```suggestion
measure. The most computationally expensive part of this procedure is the spectral decomposition of $\boldsymbol{\hat{C}}$ to find $\boldsymbol{\hat{U}}$. This can be computed exactly using an [eigensolution](#eigensolution), but an approximation of this can be computed with much lower computational expense using a [Taylor expansion](#taylorexpansion) procedure. This class provides options to perform this calculation either way, but the Taylor expansion is the default.
```

Reply:
Delete the two lines below if you like that text.